lib/puppet/util/storage.rb

```diff
@@ -53,7 +53,7 @@ def self.load
       Puppet.warning(_("Checksumfile %{filename} is not a file, ignoring") % { filename: filename })
       return
     end
-    Puppet::Util.benchmark(:debug, "Loaded state") do
+    Puppet::Util.benchmark(:debug, "Loaded state in %{seconds} seconds") do
```

Comment:
Probably worth doing translation on this, since previously we were partially translating it (even if it was only the `in %{seconds} seconds` bit)?

Reply:
It's a debug message, which means none of it should ever have been translated, but we didn't have that flexibility before.

Reply:
Fair enough. 👍